casino_classes.py

```diff
@@ -0,0 +1,61 @@
+cardsInDeck = [suit + str(val) for suit in 'cdhs' for val in [i for i in range(2, 11)] + ['j', 'q', 'k', 'a']]
+
+class Hand(object):
+    def __init__(self):
+        self._cards = {}
```

Comment:
The hands should be lists, not dicts.  Some games allow duplicates.

Reply:
ok got it, was hoping this class to handle all card games